U:RDoc::Attr[iI"uri:ETI"'Gem::RemoteFetcher::FetchError#uri;TI"RW;T:publico:RDoc::Markup::Document:@parts[o:RDoc::Markup::Paragraph;	[I"BThe URI which was being accessed when the exception happened.;T:
@fileI"#lib/rubygems/remote_fetcher.rb;T:0@omit_headings_from_table_of_contents_below0F@I"#Gem::RemoteFetcher::FetchError;TcRDoc::NormalClass0